packages/vuetify/src/components/VInfiniteScroll/VInfiniteScroll.tsx

```diff
@@ -292,6 +292,29 @@ export const VInfiniteScroll = genericComponent<VInfiniteScrollSlots>()({
         </Tag>
       )
     })
+
+    function reset () {
```

Comment:
I have played with your example - merging it with virtualized infinite scroller demo from docs, but also trying to have more realistic example, where server response decides whether there are more elements or not (typical with GraphQL offset-based pagination). Due to lack of reset, my [playground](https://play.vuetifyjs.com/playgrounds/_2AcSw) includes workaround, but it also shows that this use case would benefit from `reset()` accepting `side` as argument.

```ts
function reset(side?: InfiniteScrollSide) {
  setStatus(side ?? props.side, 'ok')
  // ...
```

Reply:
Just commited your suggestions and rebased just in case.
Let me know if I still missed anything. I double checked but since I am fairly new to the codebase I still feel like there's something that's not on my radar